### DISCUSSION TITLE
Make form field visible based on breakpoint

### DISCUSSION THREAD

**Discussion by jwhoami** [filamentphp/filament#5406] [Help]
Hi

I have the following code

`protected function getFormSchema(): array
  {
    return [
      Card::make()
        ->columns(2)
        ->schema([
          Forms\Components\TextInput::make('code')->required(),
          Forms\Components\Placeholder::make('placeholder')
            ->label(false)
            ->content('')
            ->visible(true),
          Forms\Components\TextInput::make('name')->required(),
        ])
    ];
  }`

How can I make the placeholder component only visible from the md breakpoint onwards. 

Thanks

**Comment by danharrin** [comment]
I don't think that's possible atm, but it would be a good PR! `hiddenFrom()` and `visibleFrom()` like we have for table columns

**Reply by jwhoami** [reply]
Thanks @danharrin. I'm still learning filamentphp so if someone else with more experience can handle this, it would be greatly appreciated.

Regards
